jenkinsfiles/Jenkinsfile.grobid

```diff
@@ -18,7 +18,7 @@ elifePipeline {
         stage "Build Image", {
             dir('grobid') {
                 def dockerfile = "Dockerfile"
-                if (new File("Dockerfile.crf").exists()) {
+                if (fileExists("Dockerfile.crf")) {
```

Comment:
Initially thought looking at the docs that it's supposed to be written `fileExists path/to/file`, but seems from a quick google that `fileExists(path/to/file)` also works.

Reply:
I believe the `if` statement mandates brackets but I am not so sure about the Groovy syntax either.

This answer has both versions in it: https://stackoverflow.com/a/38539009/8676953